packages/run-protocol/src/vaultFactory/liquidation.js

```diff
@@ -128,7 +130,7 @@ const makeDefaultLiquidationStrategy = amm => {
   trace(`return from makeDefault`);
 
   return {
-    makeInvitation: () => E(amm).makeSwapInInvitation(),
+    makeInvitation: _runDebt => E(amm).makeSwapInInvitation(),
```

Comment:
what motivated this?

Reply:
Accurately representing the arguments as called. But I've now changed the only caller so I should indeed remove it.